packages/react/src/components/Checkbox/index.stories.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { boolean, select, text, withKnobs } from "@storybook/addon-knobs";
+
+import Checkbox from ".";
+
+storiesOf("Form controls/Checkbox", module)
+  .addDecorator(withKnobs)
+  .addDecorator(storiesFn => (
+    <div style={{ display: "flex", justifyContent: "center" }}>
```

Comment:
Isn't it supposed to be done within the main flex decorator?
https://github.com/heetch/flamingo/blob/master/packages/react/.storybook/FlexWrapperDecorator.js

Reply:
👍

Reply:
https://retool.atlassian.net/browse/FRONT-76